### ISSUE TITLE
Occultism spirit crusher does not work unless observed

### ISSUE THREAD

**Issue by emilrasmussen1302** [FTBTeam/FTB-Modpack-Issues#4869]
### Modpack

FTB Skies Expert

### Modpack version

1.3.3

### Describe the bug

this bug is unique to servers in general, as I couldn't reproduce it in single player

### Steps to reproduce

Observing the intended speed of the crusher:
place a spirit crusher next to a pulverizer (a neutral machine to use as a time reference, placed in the same chunk) and an item collector to collect crusher drops
give a crushable item to the crusher (like raw iron) and the pulverizer (as a reference)
Observe that if you are near the spirit crusher, the crusher seems to crush at a reasonable rate, netting 8 crushed raw iron in 2:31 mins
Observing the speed of the crusher when it's not observed

### Expected behaviour

it should continue even if you are not there and it is chunkload

### Additional information

The quantum crusher phenomenon is not reproducable in single player. The same experiment setup resulted in the same crushing speed regardless of distance. The existence of the pulverizer seems to indicate that the chunk with the spirit crusher (which is the same chunk as the pulverizer one) is loaded. The spirit crusher seems to be what is malfunctioning in this case, and only due to server effects. This makes automating the crusher borderline impossible, as it require being next to it for proper function.

**Comment by jua2ja** [comment]
I originally wrote this as a bug report to the server I play in, which posted it here, so I can elaborate more on the experimental setup proving quantum crusher phenomenon: 

This is an image of the multiplayer experimental setup:
From here the spirit (djinn) crusher does seem to work
![2024-03-05_21 52 06](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/32109907/ad263def-5e3b-4d1d-9bd3-25653a2d24e6)
And from here the crusher does not seem to process any materials, but the pulverizer does work:
![2024-03-05_21 52 33](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/32109907/1c134468-77e2-491b-b67e-ca4fa51251eb)
Note that I can verify the entire experimental setup is within 1 chunk, though power is coming from outside the chunk to the pulverizer.


This is an image of the singleplayer experimental setup:
![2024-03-05_21 56 31](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/32109907/229f8d74-da92-47c6-ad63-decd9450b13a)
![2024-03-05_21 57 02](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/32109907/ae137859-f0ba-48f8-8fba-943ae50c4dbf)
Note that in single player, I did not manage to replicate this issue at all, meaning the crusher worked normally regardless of the distance I was from it. In multiplayer, I needed to be near the crusher. I don't have access to the server settings, but one of the moderators of the server is the poster of this issue.
